SparkPost/2web2ui
# SA-756 Seedlist page

This is based off #1125. Once this is merged (or Jim's PR), we can close #1125. 

How to Test:
- Start seedlist locally and point to it on openresty
- In current codebase, the endpoint is `/api/v1/inbox-placement/seedlist`; rename it to `/api/v1/inbox-placement/seeds` (according to api doc). 
- Unless you've working project, you can modify `controller.getSeedlist` (in `resources/controller.js`) to return a  list of addresses
- Set the account option (not UI option), `inbox_placement` to be `true`. 
- Start the app ~and you should see Inbox Placement -> Seed List~ and visit `/inbox-placement/seedlist` 

Notes: 
- ~Depends on https://github.com/SparkPost/accusers-api/pull/675/files to use the account option. Account option visibility is set wrong~
- ~Need to confirm path https://github.com/SparkPost/sparkpost-admin-api-documentation/pull/356#discussion_r308461529~
